crates/apub/src/http/community.rs

```diff
@@ -37,13 +37,13 @@ pub(crate) async fn get_apub_community_http(
       .await?
       .into();
 
-  if !community.deleted && !community.removed {
-    let apub = community.into_json(&context).await?;
-
-    create_apub_response(&apub)
-  } else {
-    create_apub_tombstone_response(community.actor_id.clone())
+  if community.deleted || community.removed {
```

Comment:
Instead of doing this check twice, couldn't you check the Error return type of `check_community_valid` ? Either with a match or `if let Err`

Reply:
I tried but it looks extremely messy. Better to do it this way.